src/RequestsLibrary/log.py

```diff
@@ -8,12 +8,26 @@
 
 
 def log_response(response):
+
+    try:
```

Comment:
what is the purpose of try - except? In case of timeouts?

Reply:
I see in case header is missing or malformed

Reply:
Exactly

Reply:
I think changes introduced here: #355 should do the job.